bundles/org.openhab.binding.fronius/README.md

```diff
@@ -50,30 +49,33 @@ The binding has no configuration options, all configuration is done at `bridge`,
+| Channel ID                           | Item Type            | Description                                                                                                       |
+| ------------------------------------ | -------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `inverterdatachanneldayenergy`       | Number               | Energy generated on current day                                                                                   |
+| `inverterdatachannelpac`             | Number               | AC power                                                                                                          |
+| `inverterdatachanneltotal`           | Number               | Energy generated overall                                                                                          |
+| `inverterdatachannelyear`            | Number               | Energy generated in current year                                                                                  |
+| `inverterdatachannelfac`             | Number               | AC frequency                                                                                                      |
+| `inverterdatachanneliac`             | Number               | AC current                                                                                                        |
+| `inverterdatachannelidc`             | Number               | DC current                                                                                                        |
+| `inverterdatachanneluac`             | Number               | AC voltage                                                                                                        |
+| `inverterdatachanneludc`             | Number               | DC voltage                                                                                                        |
+| `inverterdatadevicestatuserrorcode`  | Number               | Device error code                                                                                                 |
+| `inverterdatadevicestatusstatuscode` | Number               | Device status code<br />`0` - `6` Startup<br />`7` Running <br />`8` Standby<br />`9` Bootloading<br />`10` Error |
+| `powerflowchannelpgrid`              | Number               | Power (+ from grid, - to grid)                                                                                    |
+| `powerflowchannelpload`              | Number               | Power (+ generator, - consumer)                                                                                   |
+| `powerflowchannelpakku`              | Number               | Power (+ charge, - discharge)                                                                                     |
```

Comment:
Type is Number:Power
Please check the example.

Reply:
Thanks for picking this up!